server/startmonitor

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Script attempts to enable a network interface into monitor mode.
+# Note: Must be run with super user privileges
+
+
+set -e 
+
+function cleanup()
+{
+    if [[ $? -ne 0 ]]; then
+        echo -e "Usage: sudo ./startmonitor.sh [interface:mon0] [channel:3] [txpower:20]"
+    else
+        echo -e "Enabled $dev in monitor mode"
+    fi
+}
+
+# keep track of the last executed command
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
```

Comment:
Is this persistent across runs? I'm not sure what the purpose is, since this script just runs some `ip`/`iw` commands and then exits, and since nothing actually uses the value of `last_command` or `current_command`.

Reply:
just fyi, i lifted this script from https://github.com/oresat/oresat-libdxwifi/blob/master/startmonitor.sh